verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -289,37 +296,22 @@ void KytheFactsExtractor::ExtractDataTypeReference(
   }
 
   const VName type_anchor = PrintAnchorVName(type);
-  GenerateEdgeString(type_anchor, kEdgeRef, *type_vname);
+  CreateEdge(type_anchor, kEdgeRef, *type_vname);
 }
 
 VName KytheFactsExtractor::ExtractModuleInstance(
-    const IndexingFactNode& module_instance_fact_node) {
-  const auto& anchors = module_instance_fact_node.Value().Anchors();
+    const IndexingFactNode& module_instance_node) {
+  const auto& anchors = module_instance_node.Value().Anchors();
   const Anchor& instance_name = anchors[0];
 
   const VName module_instance_vname(
       file_path_, CreateScopeRelativeSignature(instance_name.Value()));
-  const VName module_instance_anchor = PrintAnchorVName(instance_name);
-
-  GenerateFactString(module_instance_vname, kFactNodeKind, kNodeVariable);
-  GenerateFactString(module_instance_vname, kFactComplete, kCompleteDefinition);
-  GenerateEdgeString(module_instance_anchor, kEdgeDefinesBinding,
-                     module_instance_vname);
-
-  // TODO(minatoma): Consider changing to children so that they can be extracted
-  // using ExtractVariableReference.
-  for (const auto& anchor :
-       verible::make_range(anchors.begin() + 1, anchors.end())) {
-    const VName* port_vname_definition =
-        vertical_scope_resolver_.SearchForDefinition(anchor.Value());
-
-    if (port_vname_definition == nullptr) {
-      continue;
-    }
 
-    const VName port_vname_anchor = PrintAnchorVName(anchor);
-    GenerateEdgeString(port_vname_anchor, kEdgeRef, *port_vname_definition);
-  }
+  const VName module_instance_anchor = PrintAnchorVName(instance_name);
```

Comment:
Recommend a few comments around these blocks, maybe a short explanation of each fact and edge w.r.t. an example.  "Given 'foo bar()', this creates facts about..."

(and generally other places too)

Reply:
I totally agree with you. But let's forget about it in this PR and comments will be added while I work on this every PR.